Source/Cogworks.Umbraco.Essentials/Builders/ResponsiveImageBuilder.cs

```diff
@@ -55,32 +55,40 @@ public class ResponsiveImageBuilder : IResponsiveImageBuilder
             int? quality = null,
             string fileExtension = null)
         {
-            breakpoints ??= BreakPointConstants.DefaultBreakpoints;
+            if (breakpoints.HasValue() && !breakpoints.BreakpointsValues.HasAny())
```

Comment:
Is logic correct here? We have value (we're not null) AND there is no values added to the BreakpointValues collection? 🤔

Reply:
This is just to make sure the initial object is not null